### ISSUE TITLE
[BUG] CPU Metrics incorrectly showing as 0

### ISSUE THREAD

**Issue by Gary-H9** [nerdswords/yet-another-cloudwatch-exporter#569]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

We are monitoring a Linux EC2 instance using the latest version of YACE. YACE appears to not be pulling CPU Utilization metrics from Cloudwatch but is pulling other metrics with respect to the same EC2 instance. 

Example: 
Cloudwatch shows data
![image](https://user-images.githubusercontent.com/26419401/165729812-60c670aa-747c-4911-887e-fd756472a9d1.png)

YACE Exported Data: 
```
aws_ec2_cpuutilization_average{account_id="<accountnumber>",dimension_AutoScalingGroupName="",dimension_ImageId="",dimension_InstanceId="i-<instancenumber>",dimension_InstanceType="",name="arn:aws:ec2:eu-west-2:<accountnumber>:instance/i-<instancenumber>",region="eu-west-2"} 0
```

Grafana showing the exported data:
![image](https://user-images.githubusercontent.com/26419401/165730287-78b942e6-e91e-43f2-a48a-18b2ad958b40.png)

We have discounted: 
- IAM permissions - these are the same as other EC2 instances which are pulling data
- grafana  (stats in Grafana were identical to raw metrics from the exporter itself)
- Added Maximum, Minimum and Sum options for the ec2cpuutilization metric to eliminate the average value as the cause. All metric statistic options showed a value of Zero.
- Port-forwarded to the cloudwatch-exporter pod (YACE). All metrics here showed zero, thus eliminating Prometheus and Grafana as the root cause.

YACE Configuration is [here](https://github.com/ministryofjustice/staff-infrastructure-monitoring-deployments/blob/main/kubernetes/infrastructure-monitoring/templates/cloudwatch-metrics/_production-metrics-custom.tpl) and [here](https://github.com/ministryofjustice/staff-infrastructure-monitoring-deployments/blob/main/kubernetes/infrastructure-monitoring/templates/cloudwatch-metrics/_production-metrics.tpl).

### Expected Behavior

We expect the metrics to be non-zero for CPU.

### Steps To Reproduce

Unsure how reproducible this is given that other EC2 instances are exporting metrics. 

To attempt to reproduce however, export CPU metrics on a linux instance using the latest YACE exporter. 

### Anything else?

This bug is being raised off of the back of [this](https://github.com/ministryofjustice/staff-infrastructure-monitoring-config/issues/234) issue.

**Comment by tonyfosterdev** [comment]
@Gary-H9 I'm seeing similar with RDS.  Did you manage to find a solution?

**Comment by Gary-H9** [comment]
👋🏻 Hey @tonyfosterdev, apologies it was some time ago and my memory is a bit hazy on it. 

My colleague put an up date [here](https://github.com/ministryofjustice/staff-infrastructure-monitoring-config/issues/234#issuecomment-1131455038). 

TLDR We weren't sure why but it started working.
